### ISSUE TITLE
App size and minSdkVersion requirement on Android

### ISSUE THREAD

**Issue by freshtechtips** [superlistapp/super_native_extensions#255]
This is yet the best plugin for clipboard for flutter as there are not many alternatives, however, the app size increases too much, is it a problem related to this plugin, or how does it work with rust?? And the requirements of minSdkVersion 23, is there any hope that it will be lower in the future?

**Comment by knopp** [comment]
> > sdkVersion 23 is Android 6.0. That's pretty old already. As for the app size increase, the binary compresses to about 500kb per platform, what kind of increase are you seeing?
> 
> Did you try the build the app and see the app size before and after adding the library and setup it up for Android?

Yes. In my experience it increases the final size of APK by about 300-600kb per platform (depending on platform), which seems reasonable considering the amount of code involved. That's why I'm asking if you're getting different numbers.

**Comment by masewo** [comment]
> sdkVersion 23 is Android 6.0. That's pretty old already. As for the app size increase, the binary compresses to about 500kb per platform, what kind of increase are you seeing?

Yes, it is pretty old, but if you have a production app with users out there, that still use Android 5.0 then you are basically locked here now.
